android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
 
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.swmansion.rnscreens">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Due to some issues with parsing in older RN CLI versions we need to restore package declaration in the manifest file.

```suggestion
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
          package="com.swmansion.rnscreens">
```